src/js/mixins/TooltipMixin.jsx

```diff
@@ -38,7 +38,8 @@ var TooltipMixin = {
   tip_handleContainerMouseMove: function (e) {
     var el = e.target;
 
-    if (el.dataset && el.dataset.behavior && el.dataset.behavior === "show-tip") {
+    if (el.dataset && el.dataset.behavior
+        && el.dataset.behavior === "show-tip") {
```

Comment:
I'd like to suggest one of:

```
if (el.dataset &&
  el.dataset.behavior &&
  el.dataset.behavior === "show-tip) {
```

or

```
if (el.dataset
  && el.dataset.behavior
  && el.dataset.behavior === "show-tip) {
```

@aldipower @philipnrmn what are your thoughts on this?